skbio/io/format/stockholm.py

```diff
@@ -0,0 +1,301 @@
+    #=GF SS    Published; PMID 9223489
+    #=GF RN    [1]
+    #=GF RM    9223489
+    #=GF RT    The role of the pseudoknot at the 3' end of turnip yellow mosaic
+    #=GF RT    virus RNA in minus-strand synthesis by the viral RNA-dependent \
+RNA
+    #=GF RT    polymerase.
+    #=GF RA    Deiman BA, Kortlever RM, Pleij CW;
+    #=GF RL    J Virol 1997;71:5990-5996.
+    AF035635.1/619-641             UGAGUUCUCGAUCUCUAAAAUCG
+    M24804.1/82-104                UGAGUUCUCUAUCUCUAAAAUCG
+    J04373.1/6212-6234             UAAGUUCUCGAUCUUUAAAAUCG
+    M24803.1/1-23                  UAAGUUCUCGAUCUCUAAAAUCG
+    #=GC SS_cons                   .AAA....<<<<aaa....>>>>
+    //
+
+Format Support
+==============
+**Has Sniffer: Yes**
+
+**State: Experimental as of 0.4.0-dev.**
+
++------+------+---------------------------------------------------------------+
+|Reader|Writer|                          Object Class                         |
++======+======+===============================================================+
+|Yes   |No    |:mod:`skbio.alignment.TabularMSA`                              |
++------+------+---------------------------------------------------------------+
+
+Format Specification
+====================
+Stockholm format contains two types of data. The first can contain raw DNA,
+RNA, or Protein data and the second is comprised of associated metadata.
+Raw data lines begin with an associated 'name', which
+often times is an id comprised of letters and numbers, such as
+'M24804.1/82-104' ([1]_). After the id and a few tab characters,
+the data is displayed. Metadata lines, however, begin with a '#' and a
+two-letter marker which describes the metadata type, for example '#=GF' (see
+Metadata Types). Each metadata line also contains an extra two-letter tag,
+such as 'AS' or 'CC' which tells what type of data it precedes. All metadata
+is optional.
+
+Metadata Types
+++++++++++++++
+GF
+--
+Data relating to multiple sequence alignment as a whole, such as authors or
+number of sequences in the alignment. Starts with #=GF followed by a tag and
+data relating to the tag. Typically comes first in a Stockholm file.
+For example:
+
+.. code-block:: none
+
+    #=GF DE CBS domain
+
+Example taken from [2]_.
+
+GS
+--
+Data relating to a specific sequence in the multiple sequence alignment.
+Starts with #=GS followed by the sequence name followed by a tag and data
+relating to the tag. Typically comes second in a Stockholm file.
+For example:
+
+.. code-block:: none
+
+    #=GS O83071/259-312 AC O83071
+
+Example taken from [2]_.
+
+GR
+--
+Data relating to the columns of a specific sequence in a multiple sequence
+alignment. Starts with #=GR followed by the sequence name followed by a tag
+and data relating to the tag. Typically comes after the data line it relates
+to.
+For example:
+
+.. code-block:: none
+
+    #=GR O31698/18-71 SS    CCCHHHHHHHHHHHHHHH..EEEEEEEE....EEEEEEEEHHH
+
+Example taken from [2]_.
+
+GC
```

Comment:
I ordered the data types in the order they usually appear in a Stockholm file, let me know if you'd rather I swap the order.